odp/identity/templates/base.html

```diff
@@ -49,6 +51,23 @@
             </div>
         </nav>
         <hr class="m-0"/>
+
+    {% elif brand == 'sadco' %}
+        <nav class="navbar navbar-expand-lg bg-white p-4">
+            <div class="container-fluid">
+                <div class="navbar-nav justify-content-start px-4 flex-grow-evenly">
+                    {{ nav_logo(image='sadco-logo.png', url='https://protosadco.saeon.ac.za') }}
```

Comment:
Remove the url from the logo (set url=none), as this would just take the user back to the site they are trying to log in from, which would be confusing.